### ISSUE TITLE
Error details in wrappers.

### ISSUE THREAD

**Issue by bhargavvader** [piskvorky/gensim#703]
Since all of the wrappers use `check_output` to run on command line, if an error does occur, there is no information apart from `subprocess.CalledProcessError` error message. The user must manually run the command on shell again to figure out what's wrong in these cases.

There should ideally be a way for the user to know what the exact nature of the error is, or at least be given instructions on how to figure out what the error is.

**Comment by piskvorky** [comment]
Good idea!

**Comment by bhargavvader** [comment]
Would like to add here that having a way to print console log/messages while running wrappers would be very useful.

**Comment by piskvorky** [comment]
Yes... especially if we can capture that output and direct it to log.

We definitely don't want to clobber our stdout with the subprocess's stdout, unless the user explicitly requests this. IMO logging is the right place for this output.

**Comment by kirit93** [comment]
To get a verbose error on the command line as you would had the command been run on the shell, the `subprocess.check_output` function does the trick if used as follows : 

```
try:
	subprocess.check_output(cmd, shell=True)
except subprocess.CalledProcessError as outerror:
	print(outerror)
```

Any particular reason a gensim `check_output` is being used instead of the `subprocess` method?

**Comment by bhargavvader** [comment]
@piskvorky , what's the reason we don't just use the `subprocess` `check_output`? Or if there was a way we could pass the `shell` flag in the current function, this would be a very useful fix.

**Comment by tmylk** [comment]
The reason for custom check_output was support for Python 2.6 which is now removed. 
We can use the  `subprocess.check_output` now

**Comment by menshikh-iv** [comment]
Will be fixed in #1501
